util/dvsim/Launcher.py

```diff
@@ -263,9 +263,14 @@ def _get_n_lines(pos, num):
                 if chk_failed:
                     if _find_patterns(self.deploy.fail_patterns, line):
                         # If failed, then nothing else to do. Just return.
+                        # For build failures show 10 extra lines in context.
+                        context = (
+                            lines[cnt:cnt+10] if self.deploy.target == "build"
+                            else []
+                        )
                         return "F", ErrorMessage(line_number=cnt + 1,
                                                  message=line.strip(),
-                                                 context=[])
+                                                 context=context)
```

Comment:
```suggestion
                                                 context=lines[-10:])
```

I think it would be better to set the context unconditionally to the last 10 lines. Coverage merge or report for example could exit if `urg` is not accessible, or if those command were buggy. Simulations runs as well, if we made a mistake and the simv is not at the right path (especially on edacloud). These lines will provide more context in those cases, rather than just plainly say "Job returned non-zero exit code".